lib/ansible/config/base.yml

```
@@ -359,7 +359,7 @@ COVERAGE_REMOTE_OUTPUT:
 COVERAGE_REMOTE_PATHS:
   name: Sets the list of paths to run coverage for.
   description:
-  - A list of paths for files on the Ansible controller to run coverage for when executing on the remote host.
+  - A list of paths for files on the Ansible controll node to run coverage for when executing on the remote host.
```

as bmillemathias said:
```suggestion
  - A list of paths for files on the Ansible control node to run coverage for when executing on the remote host.
```